### PR TITLE
Added components and modified the front end

### PR DESCRIPTION
**Pull request by Caortega4**
Abstracts the display ofj the pages

### CODE REVIEW

File: shop-local/src/components/Store-info.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
 import "../App.css"
 
-const StoreInfo = ({ name, address, description, openTime, closeTime, phoneNumber, instagram, facebook }) => {
+
+// const StoreInfo = ({ name, address, description, openTime, closeTime, phoneNumber, instagram, facebook }) => {
```
Comment (mmunozvaltierra) [inline shop-local/src/components/Store-info.js:4]: Seems like you don't need this comment block then? Would be best to take it out if not needed

File: shop-local/src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    Shop local is a platform in which you can find local businesses to support.
+                    Below are listed a couple of stores that you can try to help.
+                    This is a project by Mauricio Munoz and Carlos Ortega as part of Google's
+                    Software Product Sprint Spring 2021.
+                </p>
+            </div>
+
+            <h2>Stores:</h2>
+            <ul className='list-store'>
+                <Link className='nav-style' to='/store'>
```
Comment (mmunozvaltierra) [inline shop-local/src/components/Home.js]: Is /store a mistake? I don't see it implemented anywhere. Do we plan on adding it later

File: shop-local/src/App.js [hunk 3]
```diff
@@ -1,51 +1,57 @@
...
 function App() {
 
-  const [items] = useState([
-    { name: "Shampoo", price: "7$" },
-    { name: "Chocolate", price: "1$" },
-    { name: "Milk", price: "4$" },
-    { name: "Bitcoin", price: "999$" },
-    { name: "Candy", price: "0.5$" },
+  //   // Example of requesting a store
+  // const store = fetch("/get-store?storeName=test1")
```
Comment (mmunozvaltierra) [inline shop-local/src/App.js]: This example is in this file twice. Maybe remove one or both?

**Review by mmunozvaltierra — COMMENTED** [review]
Before approval I think the /store comment should be addressed. Seems like it can be an error

File: shop-local/src/components/Store-info.js (hunk 1 above)
Comment (Caortega4) [reply shop-local/src/components/Store-info.js:4]: This where the page for one store is created but it does now handle the json object correctly yet. Is it okay to leave it there until we get the data retrieval?

Comment (mmunozvaltierra) [reply shop-local/src/components/Store-info.js:4]: Sounds good

**Review by mmunozvaltierra — APPROVED** [review]

**Pull request merged**